dev/tidb-in-kubernetes/maintain/lightning.md

```diff
@@ -0,0 +1,139 @@
+    helm inspect values pingcap/tidb-lightning --version=<chart-version> > tidb-lightning-values.yaml
+    ```
+
+    TiDB Lightning Helm chart supports both local and remote data source.
+
+    * Local
+
+        Local mode requires the mydumper backup data to be on one of the Kubernetes node. This mode can be enabled by setting `dataSource.local.nodeName` to the node name and `dataSource.local.hostPath` to the mydumper backup data directory path which contains a file named `metadata`.
+
+    * Remote
+
+        Unlike local mode, remote mode needs to use [rclone](https://rclone.org) to download mydumper backup tarball file from a network storage to a PV. Any cloud storage supported by rclone should work, but currently only the following have been tested: [Google Cloud Storage (GCS)](https://cloud.google.com/storage/), [AWS S3](https://aws.amazon.com/s3/), [Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/).
+
+        1. Make sure that `dataSource.local.nodeName` and `dataSource.local.hostPath` are commented out.
+
+        2. Create a `Secret` containing the rclone configuration. A sample configuration is listed below. Only one cloud storage configuration is required. For other cloud storages, please refer to [rclone documentation](https://rclone.org/).
+
+            {{< copyable "" >}}
+
+            ```yaml
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: cloud-storage-secret
+            type: Opaque
+            stringData:
+              rclone.conf: |
+              [s3]
+              type = s3
+              provider = AWS
+              env_auth = false
+              access_key_id = <my-access-key>
+              secret_access_key = <my-secret-key>
+              region = us-east-1
+
+              [ceph]
+              type = s3
+              provider = Ceph
+              env_auth = false
+              access_key_id = <my-access-key>
+              secret_access_key = <my-secret-key>
+              endpoint = <ceph-object-store-endpoint>
+              region = :default-placement
+
+              [gcs]
+              type = google cloud storage
+              # The service account must include Storage Object Viewer role
+              # The content can be retrieved by `cat <service-account-file.json> | jq -c .`
+              service_account_credentials = <service-account-json-file-content>
+            ```
+
+            Fill in the placeholders with your configurations and save it as `secret.yaml`, and then create the secret via `kubectl apply -f secret.yaml -n <namespace>`.
+
+        3. Configure the `dataSource.remote.storageClassName` to an existing storage class in the Kubernetes cluster.
+
+2. Deploy TiDB Lightning
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm install pingcap/tidb-lightning --name=<tidb-lightning-release-name> --namespace=<namespace> --set failFast=true -f tidb-lightning-values.yaml --version=<chart-version>
```

Comment:
This makes it explicit for how users are using the lightning, there is no need to upgrade the lightning job, users will delete and deploy one instead of running `helm upgrade`. The existing job field can not be upgraded once created. So I think `--set` is appropriate here.

Reply:
I mean if the job is successful one should not need set

Reply:
This relies on the default value of our chart which may change at different times. Using `--set` explicitly makes clear what the command does.